### ISSUE TITLE
Add `cargo new`/`cargo init`

### ISSUE THREAD

**Issue by Ketsuban** [rust-lang-nursery/rls-vscode#204]
Right now this extension seems to assume that you already have a working project to open; there is nothing to start one easily, and if I'm going to drop to a terminal window to run `cargo init` it makes me question why I'm using a big almost-IDE like Code instead of a leaner editor like vim.

**Comment by jonathan-s** [comment]
Yeah, instead of limiting yourself to cargo new and init. I would say the issue should be: "Support cargo tasks". A good example should be all the cargo tasks that https://github.com/editor-rs/vscode-rust are currently supporting.
